Testscripts/Linux/nested_kvm_basic_test.sh

```diff
@@ -28,12 +29,20 @@
 . nested_vm_utils.sh || {
 	echo "ERROR: unable to source nested_vm_utils.sh!"
 	echo "TestAborted" > state.txt
-	exit 2
+	exit 0
 }
 
 # Source constants file and initialize most common variables
 UtilsInit
 
+case "$DISTRO" in
+	redhat_6|redhat_7)
```

Comment:
@yuxisun1217 Any reason you take out RHEL 7.x? We have been run this test on RHEL 7. last time. Also if this is the case of opt-out, then can you add centos?

Reply:
Hi @juhlee-microsoft ,
I intend to take RHEL-7 out because we won't test it in RHEL-7. Maybe the nested feature works in RHEL-7 but we won't be responsible for that. But if you want to keep this case running in RHEL-7 I can only take out RHEL-6 here. What's your opinion about that? Thanks!

Reply:
Please remove RHEL 6. I only saw 7.3 ~ the 8.x version tested in my database. It should be fine to remove RHEL 6.x

Reply:
Hi @juhlee-microsoft ,
I'm not quite sure if you understand it correctly. I just updated the script and it only skips RHEL-6 now. Thanks!